docs/support.md

```diff
@@ -0,0 +1,30 @@
+# Reliability Kit support
+
+Reliability Kit is owned and maintained by the Customer Products Reliability team at the Financial Times.
+
+It's been designed to make our user-facing systems easier to support in production environments.
+
+However, we hope that it will be useful to other teams at the FT.
+
+## The support we provide
```

Comment:
I think we could add a bit somewhere which _should_ go unsaid but I think it's worth noting: our attitude to breaking changes and migrations.

> We will not make breaking changes to Reliability Kit without a corresponding major release (e.g. `v1.0.0` to `v2.0.0`). These changes will be documented in a way that helps users migrate. We will consider the scale of these migrations and try to minimise the impact on the teams using this library and, where possible, we will mark features as deprecated some time ahead of a major release.

Reply:
Yes absolutely, thanks